workshops/views.py

```diff
@@ -521,6 +520,67 @@ def person_password(request, person_id):
     })
 
 
+@login_required
+def person_merge(request):
+    '''Merge information from one Person into another (in case of duplicates).'''
+
+    if request.method == 'POST':
+        form = PersonMergeForm(request.POST)
+        if form.is_valid():
+            # FIXME: what should be serialized?
+            request.session['person_from'] = form.fields['person_from']
+            request.session['person_to'] = form.fields['person_to']
```

Comment:
Autocompletion field yield 2 separate fields in HTML: first holds textual representation, second holds ID. The naming is `ORIGINALFIELD_0` and `ORIGINALFIELD_1`. But in the code these two separate fields aren't available.

If you want to access user-entered data after validation and parsing, use `form.cleaned_data` dict:
```python
            request.session['person_from'] = form.cleaned_data['person_from']
            request.session['person_to'] = form.cleaned_data['person_to']
```

The issue here is that `form.cleaned_data['person_from']` and `form.cleaned_data['person_to']` return Person objects, and they're not JSON-serializable.

So what I'd do is to store ID of the selected persons:
```python
            request.session['person_from'] = form.cleaned_data['person_from'].pk
            request.session['person_to'] = form.cleaned_data['person_to'].pk
```
…and retrieve they objects again in `person_merge_confirmation`:
```python
    try:
        person_from = Person.objects.get(pk=request.session.get('person_from'))
        person_to = Person.objects.get(pk=request.session.get('person_to'))
    except Person.DoesNotExist:
        messages.error(request, 'Must set both persons.')
        return redirect('person_merge')
```